src/Microsoft.AspNet.Mvc.ModelBinding/IObjectModelValidator.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.ModelBinding;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// A validator that validates a given object and adds erros to the given <see cref="ModelStateDictionary"/>.
+    /// </summary>
+    public interface IObjectModelValidator
```

Comment:
ok, the intent was renaming 
```IModelValidator``` to ```IPropertyValidator```
and ```IObjectValidator``` to ```IModelValidator```
so that ```IModelValidator``` represents the validation system and 
```IPropertyValidator``` represents the individual properties. 

Since this would require renaming lot of files, I suggest we do this as a separate change. Let me know if you disagree.